azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -379,6 +379,7 @@ private void handleExecutionJobDetailsPage(final HttpServletRequest req,
 
     page.add("projectName", project.getName());
     page.add("flowid", flow.getId());
+    page.add("flowlist", flow.getId().split(":", 0));
```

Comment:
Can we use the constant `azkaban.Constants.PATH_DELIMITER` to refer to the separator?

Reply:
Done.